auto_nag/tests/test_phonebook.py

```diff
@@ -0,0 +1,26 @@
+
+from auto_nag.scripts.phonebook import PhonebookDirectory
+
+
+class TestPhonebook:
+    def __init__(self):
+        self.demo_dict = {u'email@example.com': {
```

Comment:
I could use that, I thought, if something changed there(people.json), we are making sure that this is the format we are expecting.